rules/library.bzl

```diff
@@ -21,7 +21,7 @@ PrivateHeadersInfo = provider(
     },
 )
 
-GLOBAL_INDEX_STORE_PATH = "bazel-out/rules_ios_global_index_store.indexstore"
+GLOBAL_INDEX_STORE_PATH = "bazel-out/_global_index_store"
```

Comment:
Curious to know if anyone sees an issue with changing this?

Reply:
cc @jerrymarino do you see any issues with this?

I think I'll come up with a good way to test it and verify if it impacts existing users

Reply:
@thiagohmcruz no major issues AFAIK - I originally suggested this change the other day in our meeting and think it is the right way.